catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -48,6 +52,10 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(WorkspaceServiceImpl.class);
 
+  private static final String ID_FIELD = "id_txt";
+
+  private static final int MAX_SUBSCRIPTIONS = 100;
```

Comment:
Would it make sense to make `MAX_SUBSCRIPTIONS` configurable?

Reply:
:+1: Seems like something that would be nice to have control over without a recompile if need be.